packages/wasm_run_flutter/android/build.gradle

```
@@ -29,6 +29,8 @@ android {
     // to bump the version in their app.
     compileSdkVersion 31
 
+    namespace 'com.example.wasm_run_flutter'
```

Thanks! Maybe this would be better? or the `com.example` is used in other places?

```suggestion
    namespace 'com.wasm_run.wasm_run_flutter'
```

---

I just took the same namespace as the `group` value in Line 3 the same file.